semgrep/semgrep/target_manager.py

```diff
@@ -6,27 +6,83 @@
 from pathlib import Path
 from typing import Collection
 from typing import Dict
-from typing import FrozenSet
```

Comment:
Now everything uses `Set` for simplicity and to avoid unnecessary conversions. I wanted to use `FrozenSet` for everything but there was a set that was being constructed one element at a time and was annoying to change it.